boltdesignsystem/bolt
# [DO NOT MRGE] DS-761 replace image element in hero

## Jira

https://pegadigitalit.atlassian.net/browse/DS-761

## Summary

Removes image component from the hero component

## Details



## How to test

- Confirm no usages of the image component within the hero component
- Confirm no visual changes in the hero component examples in pattern lab
